src/test/unit_tests/test_nc_nsfs_health.js

```diff
@@ -41,6 +46,96 @@ mocha.describe('nsfs nc health', function() {
         await nb_native().fs.rmdir(DEFAULT_FS_CONFIG, bucket_storage_path);
     });
 
+    mocha.describe('nsfs nc health cli validations - should fail', function() {
```

Comment:
Not all tests are failing, suggesting editing the describe, for example:
```suggestion
    mocha.describe('nsfs nc health cli validations', function() {
```

Reply:
correct! it was supposed to be a part of the first test headline